client/my-sites/plans-features-main/hooks/use-streamlined-price-experiment.ts

```diff
@@ -18,3 +20,12 @@ export function isStreamlinedPricePlansTreatment( variationName?: string | null
 	}
 	return ! variationName.includes( 'plans_control' );
 }
+
+export function isStreamlinedPriceCheckoutTreatment( variationName?: string | null ) {
+	if ( ! variationName ) {
+		return false;
+	}
+	return (
+		! variationName.includes( 'checkout_control' ) && ! isJetpackCheckout() && ! isAkismetCheckout()
```

Comment:
Nit: Adding the Jetpack/Akismet checks means the assignment will be made but the users won't see the effects, potentially affecting the experiment results.
With the latest restriction, this is more unlikely to happen as the user would need to go through the `onboarding` flow and go through Jetpack/Akismet checkout in the same sesssion/tab. I'd still move these checks to the `isEligible` param in the `useExperiment` hook.